test/acceptance/static_page_test.rb

```diff
@@ -4,7 +4,7 @@ class StaticPageTest < AcceptanceTestCase
   def test_homepage_exists
     visit '/'
 
-    assert_content 'exercism'
+    assert_content 'Hootcode'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.